sql/core/src/test/resources/sql-tests/results/literals.sql.out

```diff
@@ -0,0 +1,148 @@
+-- Automatically generated by SQLQueryTestSuite
+-- Number of queries: 16
+
+
+-- !query 0
+select null
+-- !query 0 schema
+struct<NULL:null>
+-- !query 0 output
+NULL
+
+
+-- !query 1
+select true, false
+-- !query 1 schema
+struct<true:boolean,false:boolean>
+-- !query 1 output
+true	false
+
+
+-- !query 2
+select 1Y, 127Y, -127Y
+-- !query 2 schema
+struct<1:tinyint,127:tinyint,(-127):tinyint>
+-- !query 2 output
+1	127	-127
+
+
+-- !query 3
+select 128Y
+-- !query 3 schema
+struct<>
+-- !query 3 output
+org.apache.spark.sql.catalyst.parser.ParseException
+
+Value out of range. Value:"128" Radix:10(line 1, pos 7)
+
+== SQL ==
+select 128Y
+-------^^^
+
+
+-- !query 4
+select 1S, 32767S, -32767S
+-- !query 4 schema
+struct<1:smallint,32767:smallint,(-32767):smallint>
+-- !query 4 output
+1	32767	-32767
+
+
+-- !query 5
+select 32768S
+-- !query 5 schema
+struct<>
+-- !query 5 output
+org.apache.spark.sql.catalyst.parser.ParseException
+
+Value out of range. Value:"32768" Radix:10(line 1, pos 7)
```

Comment:
It is the error you get when calling `java.lang.Short.valueOf`. I do agree that the radix info is not that useful when you already assume this `10`.